src/OpenTelemetry.Instrumentation.AWSLambda/CHANGELOG.md

```diff
@@ -2,26 +2,33 @@
 
 ## Unreleased
 
-* Breaking change: Rename package to OpenTelemetry.Instrumentation.AWSLambda
-  (remove ".Contrib") ([#593](https://github.com/open-telemetry/opentelemetry-dotnet-contrib/pull/593))
-* Breaking change: Move public class AWSLambdaWrapper out of Implementation subnamespace
+## 1.1.0-beta2
+
+Release PR: [#590](https://github.com/open-telemetry/opentelemetry-dotnet-contrib/pull/590)
+
+This is the first release with the new package name `OpenTelemetry.Instrumentation.AWSLambda`.
+
+* BREAKING (API, behavior): Rename package to OpenTelemetry.Instrumentation.AWSLambda
+  (remove ".Contrib") ([#593](https://github.com/open-telemetry/opentelemetry-dotnet-contrib/pull/593)).
+  This also affects the `ActivitySource` name (superseding [#534](https://github.com/open-telemetry/opentelemetry-dotnet-contrib/pull/534)).
+* BREAKING (API): Move public class AWSLambdaWrapper out of Implementation subnamespace
   ([#593](https://github.com/open-telemetry/opentelemetry-dotnet-contrib/pull/593))
+* BREAKING (API): Rename overloads of `AWSLambdaWrapper.Trace` that take an async
+  handler to `TraceAsync`, to emphasize that they (usually) need to be awaited.
+  ([#608](https://github.com/open-telemetry/opentelemetry-dotnet-contrib/pull/608))
+* Rewrite of parent context handling and related changes
+  ([#408](https://github.com/open-telemetry/opentelemetry-dotnet-contrib/pull/408)):
+  * BREAKING (API): Remove `AWSLambdaWrapper.Trace` overloads
+    without `ILambdaContext` parameter.
+  * BREAKING (behavior): Add automatic parent extraction from HTTP triggers
+    (API Gateway Proxy events), using the configured global textmap propagator.
+  * BREAKING (behavior): An activity is now also created if no parent context
+    could be extracted (previously this package would only create activities if
+    a valid parent span context could be extracted with X-Ray).
+  * Add optional parent context (`ActivityContext`) to `AWSLambdaWrapper.Trace`.
```

Comment:
Also to `AWSLambdaWrapper.TraceAsync`  after renaming of async `Trace` methods.

Reply:
Done in d4127a9